### PR TITLE
doc(release): Prepare release notes for version 23.04

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Release notes 23.04

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -15,76 +15,77 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
...
+Release date: `April 26, 2023`
 
-- Compatibility with other 22.10 components.
+- Compatibility with other 23.04 components.
+- Plugins automatic installation: plugins are now installed automatically on the poller where a Host Discovery Job is scheduled. So, you won't need to install them prior to scheduling discovery jobs.
+- Job duplication: Host Discovery jobs can now be duplicated to avoid repetitive work.
+- Mapper duplication: Host Discovery Mappers can now be duplicated inside a job.
 
-#### Enhancements
+## Centreon Monitoring Connectors Manager
```
Comment (rchauvel) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:55]: @cg-tw should we not add "(formerly Plugin Packs Manager)" to the title for this release?

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 2]
```diff
@@ -11,128 +11,86 @@ Vous trouverez dans ce chapitre tout ce qui concerne **Centreon Open Source**.
...
+- [Resources Status] Both simple and forced check options are provided in Resources Status.
+- [Resources Status] Various user interface improvements in Resources Status :
+   - Aligned column contents with labels.
+   - The icon that allows you to reorder columns is now displayed only on mouseover.
+   - The columns displayed by default have been changed.
+   - Listing pagination icons are now displayed at the same time as the resource details panel.
+- [Terminology] Renamed “problems” to “alerts” in Resources Status.
+- [Terminology] Renamed “Plugin Pack” to “Monitoring Connectors” in the user interface.
+- [UI] Improved Top Counter responsiveness.
+- [UI] Added new Centreon branding.
```
Comment (rchauvel) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:48]: @cg-tw "Applied new Centreon branding" instead?

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md (hunk 1 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:55]:
```suggestion
## Centreon Monitoring Connectors Manager (formerly Plugin Packs Manager)
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 3]
```diff
@@ -11,128 +11,87 @@ Vous trouverez dans ce chapitre tout ce qui concerne **Centreon Open Source**.
...
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:48]:
```suggestion
- [UI] Applied new Centreon branding.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -16,76 +16,77 @@ If you have feature requests or want to report a bug, please contact support.
...
+Release date: `April 26, 2023`
 
-- Compatibility with other 22.10 components.
+- Compatibility with other 23.04 components.
+- Plugins automatic installation: plugins are now installed automatically on the poller where a Host Discovery Job is scheduled. So, you won't need to install them prior to scheduling discovery jobs.
+- Job duplication: Host Discovery jobs can now be duplicated to avoid repetitive work.
+- Mapper duplication: Host Discovery Mappers can now be duplicated inside a job.
 
-#### Enhancements
+## Centreon Monitoring Connectors Manager
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:56]:
```suggestion
## Centreon Monitoring Connectors Manager (formerly Plugin Packs Manager)
```

File: versioned_docs/version-23.04/releases/centreon-os.md [hunk 5]
```diff
@@ -17,124 +17,82 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Resources Status] Both simple and forced check options are provided in Resources Status.
+- [Resources Status] Various user interface improvements in Resources Status :
+   - Aligned column contents with labels.
+   - The icon that allows you to reorder columns is now displayed only on mouseover.
+   - The columns displayed by default have been changed.
+   - Listing pagination icons are now displayed at the same time as the resource details panel.
+- [Terminology] Renamed “problems” to “alerts” in Resources Status.
+- [Terminology] Renamed “Plugin Pack” to “Monitoring Connectors” in the user interface.
+- [UI] Improved Top Counter responsiveness.
+- [UI] Added new Centreon branding.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.md:49]:
```suggestion
- [UI] Applied new Centreon branding.
```

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**